.github/workflows/clients-java-build.yml

```diff
@@ -22,12 +19,18 @@ jobs:
 
     steps:
     - uses: actions/checkout@v3
+      with:
+        fetch-depth: 0    
     - name: Set up JDK
       uses: actions/setup-java@v3
       with:
         java-version: '11'
         distribution: 'temurin'
-        server-id: github
+        server-id: ossrh 
+        server-username: MAVEN_USERNAME
```

Comment:
Are these meant to be these static values and not like eg `${{ secrets.MAVEN_CENTRAL_USERNAME }}` below?

Reply:
Yes, it's the name of the env variable, declared below using `${{ secrets.MAVEN_CENTRAL_USERNAME }}`